src/helpers/initSentry.ts

```diff
@@ -0,0 +1,12 @@
+import * as Sentry from '@sentry/react'
+import env from 'helpers/env'
+
+export function initSentry() {
+  Sentry.init({
+    dsn: env.VITE_SENTRY_DSN,
+    environment: process.env.NODE_ENV,
```

Comment:
I think we don't have it in vite/browser, make sure it logs dev/prod correctly

Reply:
Works fine with https://github.com/BigWhaleLabs/ketl-invites-frontend/blob/6b9c636e9d6c81e4820828dc873194d3a89416f4/src/helpers/initSentry.ts#L7